### PR TITLE
Upgrade protocol to a new version

### PR DESCRIPTION
**Pull request by hobbydev71**

### CODE REVIEW

File: .123trigger [hunk 1]
```diff
@@ -0,0 +1,13 @@
+29
```
Comment (aliXsed) [inline .123trigger:1]: Out of curiosity what is the use of this file?

File: packages/apps-config/src/endpoints/productionRelayPolkadot.ts [hunk 2]
```diff
@@ -260,6 +260,7 @@ export const prodParasPolkadot: EndpointOption[] = [
     paraId: 2053,
     providers: {
       OmniBTC: 'wss://omnibtc.io/ws'
+      OnFinality: 'wss://nodle-parachain.api.onfinality.io/public-ws'
```
Comment (aliXsed) [inline packages/apps-config/src/endpoints/productionRelayPolkadot.ts]: This seems to be added in the wrong place, perhaps you just need to remove it.

File: packages/apps-config/src/endpoints/production.ts [hunk 3]
```diff
@@ -207,11 +207,10 @@ export const prodChains: EndpointOption[] = [
     }
   },
   {
-    info: 'nodle',
-    text: 'Nodle',
+    info: 'plasm',
+    text: 'Plasm',
```
Comment (aliXsed) [inline packages/apps-config/src/endpoints/production.ts:211]: Why Plasm is replacing Nodle?

File: package.json [hunk 4]
```diff
@@ -15,7 +15,11 @@
     "url": "https://github.com/polkadot-js/apps.git"
   },
   "sideEffects": false,
-  "version": "0.116.1",
+  "version": "0.118.1",
```
Comment (aliXsed) [inline package.json:18]: Can we change the version to "0.118.1-n0"? "n can stand for Nodle as the flavour of this version and "0" after "n" means the first nodle version based on 0.118.1.

File: package.json [hunk 5]
```diff
@@ -15,7 +15,11 @@
...
+  "versions": {
+    "git": "0.118.1",
+    "npm": "0.118.1"
+  },
```
Comment (aliXsed) [inline package.json:22]: Do we need the git and npm versions?

File: .123trigger (hunk 1 above)
Comment (hobbydev71) [reply .123trigger:1]: It seems that had been written the upgrade versions.

File: package.json (hunk 5 above)
Comment (hobbydev71) [reply package.json:22]: No, we don't need. I will remove them.

File: package.json (hunk 4 above)
Comment (hobbydev71) [reply package.json:18]: Yeah, got it.

File: packages/apps-config/src/endpoints/production.ts (hunk 3 above)
Comment (hobbydev71) [reply packages/apps-config/src/endpoints/production.ts:211]: We already added the nodle on top.
So we need to remove it on production.

File: package.json [hunk 6]
```diff
@@ -15,7 +15,11 @@
     "url": "https://github.com/polkadot-js/apps.git"
   },
   "sideEffects": false,
-  "version": "0.116.1",
+  "version": "0.118.1-n0",
+  "versions": {
+    "git": "0.118.1",
+    "npm": "0.118.1"
+  },
```
Comment (aliXsed) [inline package.json:22]:
```suggestion
```

**Review by aliXsed — APPROVED** [review]

**Review by Douglasacost — APPROVED** [review]
LGTM, just reviewed @hobbydev71 commits and ignore the previous ones 👍

**Pull request merged**